core/new-gui/src/app/workspace/component/result-panel/console-frame/console-frame.component.ts

```diff
@@ -117,6 +129,27 @@ export class ConsoleFrameComponent implements OnInit, OnChanges {
 
   displayConsoleMessages(operatorId: string): void {
     this.consoleMessages = operatorId ? this.workflowConsoleService.getConsoleMessages(operatorId) || [] : [];
+    console.log("got messages", this.consoleMessages);
+  }
+
+  submitCommand() {
+    if (isDefined(this.operatorId)) {
+      let workers = [];
+      if (this.targetWorker === "All Workers") {
+        workers = [...this.workerIds];
+      } else {
+        workers.push(this.targetWorker);
+      }
+      for (let worker of workers) {
+        this.workflowWebsocketService.send("PythonDebugCommandRequest", {
```

Comment:
We are kind of breaking up this request in the frontend, rather in the backend. I think either way is fine for now.

Reply:
I agree either way is fine. I prefer separate for now.